### ISSUE TITLE
Cannot reach the reported performance

### ISSUE THREAD

**Issue by Reply1999** [OrangeYHChen/TIPCB#3]
Thanks for your code!

I run the code using the default configurations on cuda=10.2 and pytorch=1.9.0, and only obtain the following results:
Rank@1:0.626545 Rank@5:0.825960 Rank@10:0.885491 mAP:0.534096.
Is it due to the cuda and pytorch version?

The detailed configurations are as follows:
CMPM: true
adam_alpha: 0.9
adam_beta: 0.999
adam_lr: 0.003
batch_size: 64
checkpoint_dir: ./log
dataset: CUHKPEDES
dir: ./datasets/
droprate: 0
embedding_type: BERT
epoches_decay: '50'
epsilon: 1.0e-08
feature_size: 2048
gpus: '1'
height: 384
log_dir: ./log
lr_decay_ratio: 0.1
lr_decay_type: MultiStepLR
max_length: 64
name: Origin
num_classes: 11003
num_epoches: 10000000
optimizer: adam
pretrained: true
resume: false
seed: 0
warm_epoch: 10
wd: 4.0e-05
width: 128

**Comment by OrangeYHChen** [comment]
I think the version difference will have a great impact. We initially train our framework on pytorch 1.1. We re-run our code with pytorch 1.8 and pytorch 1.7, and get two different results. We get Rank-1 = 62.72 on pytorch 1.8 and Rank-1 = 62.66 on pytorch 1.7. However, by further optimizing this framework, we have achieved Rank-1 = 65.39 on pytorch 1.8, and we will release the new version in the future.

**Comment by Reply1999** [comment]
Ok, thanks for your quick reply.
